include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -206,6 +344,17 @@ class SplineEvaluator
                 });
     }
 
+    /** @brief Perform batched 1D integrations of a spline function (described by its spline coefficients) along the dimension of interest and store results on a subdomain of batch_domain.
```

Comment:
```suggestion
    /** @brief Perform batched 1D integrations of spline functions (described by their spline coefficients) along the dimension of interest and store results on a subdomain of batch_domain.
```

Reply:
What I designate here with "spline function" is a multidimensional function which is defined on B-splines along dimension of interest and batch dimensions on batch domain. This is why in 1D methods I make a distinction with "1D spline function". Maybe I can add an explaination somewhere, but changing this would require a lot of change for consistency.

Reply:
> Maybe I can add an explaination somewhere

That would be great. I think it is needed as this "multidimensional function" is discrete in most of its dimensions but continuous in one. That is an odd convention which should be explained.

Reply:
I precised it with " spline function defined on a cartesian product of batch_domain and B-splines"